pennylane/qchem/tapering.py

```diff
@@ -541,55 +541,108 @@ def taper_hf(generators, paulixops, paulix_sector, num_electrons, num_wires):
     return np.array(tapered_hartree_fock).astype(int)
 
 
+def _is_callable_operation(operation, op_wires=None, op_gen=None):
+    r"""Checks whether the operation and op_gen are callables. If yes, it will return their respective objects
+    instantiated with dummy argument values required to obtain their generators and a flag value to represent
+    if such an instantiation has been done.
+
+    Args:
+        operation (Operation or Callable): qubit operation to be tapered, or a function that applies that operation
```

Comment:
Why Callable not just Operation?

Reply:
So that we can support tapering for `qml.RX` in addition to `qml.RX(0.6)`. For the former case, we build an operation ourselves with a dummy parameter value to get the required generator.